test/memory.carp

```diff
@@ -231,11 +231,11 @@
 
 (defn array-maximum []
   (let [xs [1 7 2 9 3 8 4 6 5]]
-    (assert (= 9 (maximum &xs)))))
+    (assert (= (Maybe.Just 9) (maximum &xs)))))
```

Comment:
We’ll have to use `(= &(Maybe.Just 9) &(maximum &xs))` here, because comparison is implemented for refs of `Maybe`